django/template/engine.py

```diff
@@ -58,7 +58,8 @@ def get_default():
         """
         When only one DjangoTemplates backend is configured, return it.
 
-        Raise ImproperlyConfigured otherwise.
+        If multiple DjangoTemplates backends are configured, returns the first
```

Comment:
Phrase that imperatively ('return' instead of 'return**s**'), analogous to the [change you supersede here](https://github.com/django/django/commit/4696078832f486ba63f0783a0795294b3d80d862#diff-2ec90d8b6d7f44124689729e42876209R61).